client/src/app/components/page/task-detail/task-detail.component.html

```diff
@@ -1,27 +1,35 @@
-<div class="card mt-4 container flex-row">
+<div class="mt-4 container border rounded-lg">
     <div class="row">
         <div class="colCard col-sm">
             <form [formGroup]="taskForm" (submit)="onSubmit()">
                 <h5 class="font-weight-light text-center text-lg-left mt-2 mb-2">Title</h5>
                 <input formControlName="title" class="form-control">
                 <h5 class="mt-2">Description</h5>
                 <input formControlName='description' class="form-control"><br>
-                <button class="btn btn-outline-success my-2 my-sm-0 w-50">
-                    {{buttonText}}</button>
             </form>
```

Comment:
The form should wrap all form elements, like buttons, pickers. This way you can use form events just by setting button type to `submit`. Please extend form to wrap all elements.

Reply:
Now form contains pickers and buttons. Also added types to inputs.